### PR TITLE
Adds html pages based on wireframes sketches

### PR DESCRIPTION
**Pull request by JenNailsit**

### CODE REVIEW

File: LowerSchool.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Document</title>
```
Comment (axamei) [inline LowerSchool.html:8]: I recommend changing the title of the pages.

File: LowerSchool.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  <li>2s-3s</li>
+  <li>Junior Kindergarten</li>
+  <li>Kindergarten</li>
+  <li>1st grade</li>
+  <li>2nd grade</li>
+  <li>3rd grade</li>
+  <li>4th grade</li>
+  <li>5th grade</li>
+
+</body>
```
Comment (axamei) [inline LowerSchool.html:38]: The closing `</body>` and `</html>` tags should be the last two lines in your html files.

File: LowerSchool.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      <li>
+        <a href="index.html">Home</a>
+      </li>
+      <li>
+        <a href="lowerschool.html">Lower School</a>
+      </li>
+      <li>
+        <a href="upperschool.html">Upper School</a>
+  </nav>
+  <ul></ul>
```
Comment (axamei) [inline LowerSchool.html:27]: The closing `</ul>` tag should be on line 33 after the last list item.

File: LowerSchool.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  <nav>
+    <ul>
+      <li>
+        <a href="index.html">Home</a>
+      </li>
+      <li>
+        <a href="lowerschool.html">Lower School</a>
+      </li>
+      <li>
+        <a href="upperschool.html">Upper School</a>
```
Comment (axamei) [inline LowerSchool.html:21]: This list item is missing a `</li>` and the list is missing a `</ul>`

File: LowerSchool.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  <li>3rd grade</li>
+  <li>4th grade</li>
+  <li>5th grade</li>
+
+</body>
+
+</html>
+
+<h1>Lower School</h1>
+<h6>The Lower School includes the preschool and the elemntary grades, 18 months through 5th grade.</h6>
```
Comment (axamei) [inline LowerSchool.html]: You should not skip heading levels. This should be an `<h2>` instead. You can adjust the sizing of the font with CSS.

File: LowerSchool.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+<figure>
+
+</figure>
```
Comment (axamei) [inline LowerSchool.html:46]: This closing `</figure>` tag should be on line 46 after the figcaption.

File: LowerSchool.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+<img src="images/LSimage1.png" alt="Photo of LS student." />"
+<figcaption>After reading TMNT: Friends till the End </figcaption>
+
+<figure>
+
+</figure>
```
Comment (axamei) [inline LowerSchool.html:52]: This closing `</figure>` tag should be on line 52 after the figcaption.

File: LowerSchool.html [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+<img src="images/LSimage2.png" alt="Photo of LS student." />"
+<figcaption>Podcasting</figcaption>
+
+<figure>
+
+</figure>
```
Comment (axamei) [inline LowerSchool.html:58]: This closing `</figure>` tag should be on line 58 after the figcaption.

File: UpperSchool.html [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+  <nav>
+    <ul>
+      <li>
+        <a href="index.html">Home</a>
+      </li>
+      <li>
+        <a href="lowerschool.html">Lower School</a>
+      </li>
+    </ul>
+        <a href="upperschool.html">Upper School</a>
```
Comment (axamei) [inline UpperSchool.html:21]: This list item should appear before the closing `</ul>` tag and needs to be inside a `<li>` tagset

File: UpperSchool.html [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+  </nav>
+  <ul></ul>
```
Comment (axamei) [inline UpperSchool.html:23]: The closing `</ul>` tag should be placed after the last list item of the list.

File: UpperSchool.html [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+  <li>6th-8th grade</li>
+  <li>9th-12th grade</li>
+</body>
```
Comment (axamei) [inline UpperSchool.html]: Move the `</body>` and `</html>` tags to the end of the file.

**Review by axamei — CHANGES_REQUESTED** [review]
There are some repeated errors across the html files that I would like you to make changes to. I did not mark all of them, but you should be able to see a pattern of what needs changing. 

In addition to the changes requested, I recommend adding a `<header>`, `<main>`, and `<footer>` element with the appropriate elements nested inside those tags. This will create more meaningful structure for your pages.

File: LowerSchool.html (hunk 1 above)
Comment (JenNailsit) [reply LowerSchool.html:8]: Dear Axa, I am hoping that my request for your review has gone through. I realize that I am behind and am doing my best. Thank you, Jen

**Review by JenNailsit — COMMENTED** [review]
Dear Axa, I am hoping that my request for your review has gone through. I realize that I am behind and am doing my best. Thank you, Jen

**Review by axamei — APPROVED** [review]

**Pull request merged**